auth/proofOfPossession_test.go

```diff
@@ -0,0 +1,190 @@
+// SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Cloud Security Client Go contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+package auth
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/base64"
+	"encoding/pem"
+	"github.com/lestrrat-go/jwx/jwt"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"math/big"
+	"os"
+	"path"
+	"testing"
+)
+
+func TestProofOfPossession_ParseCertHeader_edgeCases(t *testing.T) {
```

Comment:
What means edge cases?

Reply:
some specific cases which are not covered by the table tests